src/foam/core/Currency.js

```diff
@@ -129,7 +129,7 @@
     },
     {
       name: 'format',
-      code: function(amount, hideId) {
+      code: function(amount, hideSymbol, hideId) {
```

Comment:
If you made hideSymbol the third argument, it would still be compatible with the previous signature.